### ISSUE TITLE
Token with lights causes lighting to show through to other floors

### ISSUE THREAD

**Issue by teroparvinen** [theripper93/Levels#80]
**ALL OTHER MODULES DISABLED?** true
**Module:** Levels 2.5
**Foundry and System version**: Foundry version 9 build 249, DnD5e 1.5.7
**Short Description of bug**: Normal lights placed on levels work as expected. Tokens with lights enabled cause lighting, but this lighting bleeds through to other levels.

**Simple steps to reproduce the bug**:
- Empty scene with a base bitmap, walled off at the edges
- Create two levels: 0–9, 10–19
- Add one tile representing the 10–19 level on that level
- Draw simple rectangular walls containing the upper level and do the same on the lower level
- Place a light in the upper right hand corner on the upper level and another light in the lower left hand corner of the lower level
- Make a test token (with no lights enabled) inside the "building", confirm lights are working properly by toggling the token elevation between 0 and 10
- Make another test token on the lower level, give it a light radius
- Switch to the original test token, toggle elevation between 0 and 10. The second token's light is showing through on both levels
- Hide the second token and repeat, everything working properly again

**Screenshots and/or console errors**:

Lower floor with basic light
![Lower floor with basic light](https://user-images.githubusercontent.com/1235386/154512789-2ac23793-9e50-49be-a581-ee20a37a1649.png)

Upper floor with basic light
![Upper floor with basic light](https://user-images.githubusercontent.com/1235386/154512626-ad561409-a935-43ab-91b0-be0ade41bde0.png)

Lower floor with additional light on a token
![Lower floor with additional light on a token](https://user-images.githubusercontent.com/1235386/154512924-4ecff268-6bc1-48ff-8780-66c074726fef.png)

Upper floor with the light on the additional token on the lower floor showing through
![Upper floor with the light on the additional token on the lower floor showing through](https://user-images.githubusercontent.com/1235386/154513068-65e9f28e-c049-4fdd-bb4e-3af8c2c13c33.png)

**Comment by theripper93** [comment]
i tryed your exact steps and cannot seem to repro the issue unfortunatelly, any other information that could be missing?

**Comment by teroparvinen** [comment]
I uploaded the entire world I tested this on.

[test.zip](https://github.com/theripper93/Levels/files/8094363/test.zip)

The "Guard" token is the light source causing the issue, the "Test" token is the one observing the issue at +10 ft.

I honestly can't think of anything else to say because the entire world/scene is as basic as I could make it, but I see the exact same with more complex setups. I'm running in the embedded Foundry browser, but a test client connecting using chrome can see the same thing with player privileges. Also, I'm running on a Mac, but this doesn't really seem like something that would be affected by that.
